code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -384,8 +386,8 @@
 	SEND_SIGNAL(user, COMSIG_XENO_TURF_CLICK_CTRL, src)
 	..()
 
-//Pick up monkey
-/mob/living/carbon/monkey/CtrlClick(mob/user)
+//Pick up monkey, doing this on the monkey helper subtype is not really kosher, but neither are these fucking stupid signals. use the fucking base signals you clowns.
```

Comment:
This should probably be going other way, as in console subscribing to the users clicks.